confluent_kafka/admin/__init__.py

```diff
@@ -478,8 +433,7 @@ def alter_configs(self, resources, **kwargs):
                   the provided resources with the new configuration given,
                   reverting all other configuration for the resource back
                   to their default values.
-                  Use incremental=True to change the behaviour so that only the
-                  passed configuration is modified, requires broker version with KIP-248 support.
+                  If and when KIP-248 is adopted incremental updates will be possible.
```

Comment:
Remove, this will not be part of KIP-248.

Reply:
ack